src/tests/ftest/control/daos_server_helper.py

```diff
@@ -34,17 +36,23 @@ def test_daos_server_helper_format(self):
         :avocado: tags=DaosPrivHelperTest,test_daos_server_helper_format
         """
         # Verify that daos_server_helper has the correct permissions
-        self.log.info("Checking daos_server_helper binary permissions")
-        file_stats = os.stat("/usr/bin/daos_server_helper")
+        # Get the result remotely with os.stat so the format is compatible with local code
+        self.log_step("Verify daos_server_helper binary permissions")
+        helper_path = os.path.join(self.prefix, "bin", "daos_server_helper")
+        cmd = f"python3 -c 'import os; print(os.stat(\"{helper_path}\").st_mode)'"
+        result = run_remote(self.log, NodeSet(self.hostlist_servers[0]), cmd)
```

Comment:
Should we limit this to one server or run it on all server hosts?

Reply:
I guess we might as well run it on all and enforce it is consistent and correct.